setup-certs.sh

```diff
@@ -15,7 +15,7 @@ SERVER_CSR="${LOCATION}/server.csr"
 echo $SERVER_KEY, $SERVER_CERT, $DHPARAM, $CA_KEY
 
 printf "# Create new dhparam. This may take a few minutes...\n"
-openssl dhparam -out $DHPARAM 2048
+openssl dhparam -out $DHPARAM 128
```

Comment:
These certs are fake in a way - a fake cert authority is created - so I reduced this down to 128 to make it quick since people shouldn't use these in real life anyway.